src/main/java/com/auth0/json/mgmt/authenticationmethods/AuthenticationMethod.java

```diff
@@ -0,0 +1,247 @@
+package com.auth0.json.mgmt.authenticationmethods;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AuthenticationMethod implements Serializable {
```

Comment:
no need to implement Serializable. Looks like we have it in some old classes but is not needed.